go.mod

```diff
@@ -62,7 +62,6 @@ require (
 	golang.org/x/lint v0.0.0-20190409202823-959b441ac422
 	golang.org/x/sys v0.0.0-20190801041406-cbf593c0f2f3 // indirect
 	golang.org/x/tools v0.0.0-20190924052046-3ac2a5bbd98a
-	google.golang.org/appengine v1.4.0 // indirect
```

Comment:
?

Reply:
just one misc commit to remove the unused lib.